notebooks/245-oneformer-segmentation/245-oneformer-segmentation.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<p>Here is ONNX mentioned again</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>